lib/compilers/glimmer/index.js

```diff
@@ -12,13 +12,28 @@ export default class GlimmerTemplatePrecompiler extends Filter {
   }
 
   processString(content, relativePath) {
-    let specifier = getTemplateSpecifier(this.rootName, relativePath);
-    return `export default ${this.precompile(content, { meta: { specifier, '<template-meta>': true } })};`;
+    let specifier = getTemplateSpecifier(this.rootName, removeManagerIdFromPath(relativePath));
+    let managerId = getManagerIdFromPath(relativePath);
+    let meta = { specifier };
+
+    if (managerId) {
+      meta.managerId = managerId;
+    }
+
+    return `export default ${this.precompile(content, { meta })};`;
   }
 
   precompile(content, options) {
     return precompile(content, options);
   }
+
+  getDestFilePath(relativePath) {
+    let processedPath = super.getDestFilePath(relativePath);
+
+    if (!processedPath) return processedPath;
+
+    return removeManagerIdFromPath(processedPath);
```

Comment:
Why do it here? Or why do it at all? I'll answer both because I'm unsure of which you're asking. Why here: because this is where we prepare templates for consumption. It seemed like a natural place to put it; taking the managerId out of the filename and putting it into the template meta in one swoop. Why at all: Because `import foo from 'some/path/template.fragment';` didn't seem to work in the module map. It's possible I had something else wrong going on though?